libraries/ESP8266WiFi/src/ESP8266WiFiSTA.cpp

```diff
@@ -571,6 +571,8 @@ int32_t ESP8266WiFiSTAClass::RSSI(void) {
 // -------------------------------------------------- STA remote configure -----------------------------------------------
 // -----------------------------------------------------------------------------------------------------------------------
 
+#ifdef NO_EXTRA_4K_HEAP

```

Comment:
Please add a descriptive comment about what the #define means, or at least reference the issue.